tests/zeroconfig/windows/README.md

```diff
@@ -0,0 +1,19 @@
+# Tests for Splunk Zero Configuration of .NET applications hosted on Windows IIS
+
+Tests in this directory validate that automatic instrumentation of
+.NET and .NET Framework applications hosted on Windows IIS are working under
+a deployment following the Zero Configuration procedures.
+
+## Requirements to run the tests locally
+
+The GitHub job `windows-zeroconfig-e2e-test` in the [`build-and-test`](../../../.github/workflows/build-and-test.yml)
```

Comment:
Having all the setup in ci logic can make maintainability and debugging difficult. Would it be reasonable to migrate to a ps1 script and invoke there?

Reply:
I personally prefer the separate action steps: typically you run a few of them and then repeat a specific one to debug/investigate. My guess is that the script will be run only in CI, for debug/investigation devs will copy and paste particular lines of the script. That said I will make the change so we can better judge it.

Reply:
After writing it, I liked it :) - let's see if it passes CI ...

Reply:
Thanks!